### PR TITLE
Refactor application to look similar to gaiad code, add makefile commands

### PR DESCRIPTION
**Pull request by Anmol1696**
Update the core one code to look similar to gaiad to achieve:
* Easier code reading, making code more golang friendly
* Future upgrades become easier due to code compatibility
* Update make commands to match upstream

## Future work
Still needs better testing, unable to verify if the current changes are not breaking. Opening PR for more conversations.

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -52,23 +50,29 @@ import (
 	sdkCrisisKeeper "github.com/cosmos/cosmos-sdk/x/crisis/keeper"
 	sdkCrisisTypes "github.com/cosmos/cosmos-sdk/x/crisis/types"
 	"github.com/cosmos/cosmos-sdk/x/distribution"
+	distributionClient "github.com/cosmos/cosmos-sdk/x/distribution/client"
 	sdkDistributionKeeper "github.com/cosmos/cosmos-sdk/x/distribution/keeper"
+	distributionTypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
 	sdkDistributionTypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
```
Comment (puneet2019) [inline app/app.go:56]: Duplicate import, this should have been checked by linter

File: app/app.go [hunk 2]
```diff
@@ -78,26 +82,31 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/staking"
 	sdkStakingKeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
 	sdkStakingTypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	stakingTypes "github.com/cosmos/cosmos-sdk/x/staking/types"
```
Comment (puneet2019) [inline app/app.go:85]: Duplicate import

File: app/app.go [hunk 3]
```diff
@@ -106,6 +115,52 @@ import (
 	tendermintProto "github.com/tendermint/tendermint/proto/tendermint/types"
 	tendermintDB "github.com/tendermint/tm-db"
 	"honnef.co/go/tools/version"
+
+	applicationParams "github.com/persistenceOne/persistenceCore/app/params"
```
Comment (puneet2019) [inline app/app.go:119]: rename to appParams?

File: cmd/persistenceCore/cmd/root.go [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+		traceStore,
+		loadLatest,
+		uint(1),
+		map[int64]bool{},
+		homePath,
+		appOpts,
+	)
+
+	if height != -1 {
+		if err := gaiaApp.LoadHeight(height); err != nil {
```
Comment (puneet2019) [inline cmd/persistenceCore/cmd/root.go:278]: not gaiaApp

Comment (puneet2019) [reply cmd/persistenceCore/cmd/root.go:278]: can you take this up as well, local variable

Comment (Anmol1696) [reply cmd/persistenceCore/cmd/root.go]: done

**Review by puneet2019 — APPROVED** [review]
LGTM

**Pull request merged**